packages/api/src/index.ts

```diff
@@ -109,6 +120,20 @@ export default async function makeApp(params: CliArgs) {
   app.use(router);
 
   if (listen) {
+    let version = new Gauge({
+      name: "livepeer_api_versions",
+      help: "Versions used by livepeer api node",
+      labelNames: ["nodeversion", "apiversion", "os", "arch"],
+    });
+    version.set(
+      {
+        nodeversion: process.version,
+        arch: process.arch,
+        os: process.platform,
+        apiversion: getGitHash(),
```

Comment:
I wonder if this should be just a global field in all metrics instead, like configured on the Prometheus client. Then we could also compare any metric based on the app version. It should not increase the cardinality, but not sure if it'd increase storage use significantly (victoria compresses the hell out of it tho, maybe not).

Also, I'd name this just `version`, as `www` doesn't really exist in the API server context. WDYt?